test/_generate_expected_node.js

```diff
@@ -1,30 +1,85 @@
-const canvg = require("../src/canvg.js"),
-  Canvas = require("canvas"),
-  path = require("path"),
-  Promise = require("bluebird"),
-  fs = Promise.promisifyAll(require("fs")),
-  PNGImage = Promise.promisifyAll(require("pngjs-image"));
+const canvg = require('../src/canvg.js'),
+  Canvas = require('canvas'),
+  path = require('path'),
+  Promise = require('bluebird'),
+  fs = Promise.promisifyAll(require('fs')),
+  PNGImage = Promise.promisifyAll(require('pngjs-image'));
 
-const file = process.argv[2];
-const fileName = path.resolve(`${__dirname}/../svgs/${file}`);
-if (!fs.existsSync(fileName)) {
-  throw new Error(`${fileName} does not exist!`);
+function createCanvas (width, height) { // 2.0 API
+  return new Canvas(width, height);
 }
 
+const file = process.argv[2];
+const start = process.argv[3] || 0;
+const end = process.argv[4];
```

Comment:
it seems you can still just all this with `file.svg` and it'll work as expected, but you added more options for processing in batch, correct?

Reply:
Right.